lib/supa/command.rb

```diff
@@ -16,8 +16,12 @@ def represent
     private
 
     def value
-      return @representer.send(@options[:modifier], apply_render_flags(raw_value)) if @options[:modifier]
-      apply_render_flags(raw_value)
+      return @value if defined?(@value)
+      @value = modifier ? @representer.send(modifier, apply_render_flags(raw_value)) : apply_render_flags(raw_value)
```

Comment:
Maybe then let's have an agreement about modifier

1. It's a symbol
2. It's a lambda/proc
3. Exception otherwise

Then it will be super straightforward